esbuilder/lit-connect-modal/src/modal.js

```diff
@@ -0,0 +1,163 @@
+import MicroModal from 'micromodal';
+import css from './modal.css';
+import {
+  rawListOfWalletsArray,
+  metaMaskSingle,
+  coinbaseSingle,
+} from './helpers/walletList.js';
+import providerMethods from './helpers/providerMethods.js';
+
+export default class LitConnectModal {
+  constructor({ providerOptions }) {
+    this.dialog = MicroModal;
+    this.closeAction = undefined;
+    this.parent = document.body;
```

Comment:
same as above should we pass the parent tag as a param?

Reply:
same as above

Reply:
Can we create a Linear ticket to track this?